infra/bin/test-ci.ts

```diff
@@ -34,13 +34,13 @@ new TestCIStack(app, "TestCIStack", {
     appWebhookSecret:
       app.node.tryGetContext("app_webhook_secret") ||
       process.env.GITHUB_APP_WEBHOOK_SECRET,
+    hostedZoneId:
+      app.node.tryGetContext("hosted_zone_id") || process.env.HOSTED_ZONE_ID,
   },
   logsBucketName:
     app.node.tryGetContext("logs_bucket") || process.env.LOGS_BUCKET,
   logsBucketImport:
     app.node.tryGetContext("logs_bucket_import") ||
     process.env.LOGS_BUCKET_IMPORT ||
     false,
-  pvreBucketName:
```

Comment:
Is this no longer used anywhere?

Reply:
The SSM parameters that we used to have in here was not being used - we removed the `DaemonSet` that we used for that a long time ago. It wasn't compatible with the EKS blueprints configuration anyway so I have stripped it here.